geonode/monitoring/collector.py

```diff
@@ -452,7 +451,7 @@ def _key(v):
                                   'label': label,
                                   'samples_count': samples,
                                   'value_raw': value or 0,
-                                  'value_num': value if isinstance(value, integer_types + (float, Decimal,)) else None})
+                                  'value_num': value if isinstance(value, int + (float, Decimal,)) else None})
```

Comment:
`integer_types` is a collection/sequence so `integer_types + (float, Decimal)` extends the sequence. `int` is not a sequence and does not support the `+` operation on the `tuple` type. Please fix this to simply contain `int` within the `tuple`.